app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import (NotVaccinatedError, NotWearingMaskError,
+                        OutdatedVaccineError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError()
+
+        expiration_date = visitor["vaccine"]["expiration_date"]
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError()
```

Comment:
Consider providing a descriptive message when raising the `OutdatedVaccineError` to comply with the checklist requirement for descriptive error messages. Example: `raise OutdatedVaccineError('Visitor's vaccine is outdated.')`.